Libraries/LibJS/Tests/functions/function-name.js

```diff
@@ -48,3 +48,10 @@ test("names of native functions", () => {
     expect((console.debug.name = "warn")).toBe("warn");
     expect(console.debug.name).toBe("debug");
 });
+
+test("cyclic members should not cause infinite recursion (#3471)", () => {
+    let a = [()=>4];
```

Comment:
please format tests with prettier :)

*probably:*

```suggestion
    let a = [() => 4];
```

Reply:
Some day I'll remember about this :P